pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -812,7 +812,7 @@ protected boolean isNamespaceReplicated(NamespaceName namespaceName) {
         try {
             String topicPartitionPath = joinPath(MANAGED_LEDGER_PATH_ZNODE,
                     namespaceName.toString(), topicDomain.value());
-            List<String> topics = globalZk().getChildren(topicPartitionPath, false);
+            List<String> topics = localZk().getChildren(topicPartitionPath, false);
```

Comment:
Is it possible to add a unit test? This will make pulsar more robust

Reply:
There is  already has a test `AdminApiTest2#testMaxTopicsPerNamespace`

Reply:
Thanks @aloyszhang , This change lgtm. As @codelipenghui mentioned, it would be great if we have it reproduced and fixed in the unit test.  
How about we create 2 mocked zk in the `MockedPulsarServiceBaseTest`, 1 zk for localZk() and another for globalZk? By this way, we could reproduce this error in the original code, and verify it fixed by this change.

Reply:
Thanks for yours suggestion @jiazhai @codelipenghui .
I have test the `AdminApiTest2#testMaxTopicsPerNamespace` and found the globalZk and localZk return the same result , this is why the original test passed.
I'll try to add two different zk for local and global respectively.